### ISSUE TITLE
M365 LP failed on a Tenant where we had O365 LP installed

### ISSUE THREAD

**Issue by AchimBossler** [SharePoint/sp-provisioning-service#262]
Correlation ID: e2e144a9-af30-4907-a8bb-337470041096 

On this tenant there was an installation of the previous version (O365 LP). We deleted the O365LP-Site Collection and we removed the app in the App Catalogue before we started the provisioning of M365 LP. The Default Language of the tenant is English.

**Comment by PaoloPia** [comment]
Hi,
Thanks for your feedback. Looking into the log, we can see that the exception was "An unexpected error occurred while manipulating the navigational structure of this Web.". It looks like an unexpected hiccup. Could you please try again, and in case of any further exception create another issue here? In the meantime, I will close this item. Thanks.
